src/ogc-api/endpoint.ts

```diff
@@ -405,7 +421,8 @@ ${e.message}`);
     skipGeometry: boolean = null,
     sortby: string[] = null,
     bbox: [number, number, number, number] = null,
-    properties: string[] = null
+    properties: string[] = null,
+    datetime: {start?: Date, end?: Date} = null
```

Comment:
How about adding such a type in `model.ts`:

```ts
export type DateTime = { start: Date } | { end: Date } | {start: Date, end: Date }
```

This way the typing will prevent a parameter that has neither `start` nor `end` date set.